PraceDomowe/PracaDomowa5/Mach_Aleksandra/Aleksandra Mach Praca domowa 5.ipynb

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
<p>Liczba takich postów została policzona poprawnie (1039), jednak liczba pytań to jest liczba postów, dla których <code>PostTypeId</code>==2 (chociaż to trzeba by było odczytać z opisu danych).</p><p> </p><p><code>2 punkty</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>